tests/test_export.py

```diff
@@ -274,7 +274,7 @@ def test_export_bad_modifier(caplog):
     "spec, normfactor_config",
     [
         (spec_staterror(), dict(name='mu', inits=[1.0], bounds=[[0.0, 8.0]])),
-        (spec_histosys(), dict()),
+        (spec_histosys(), {}),
```

Comment:
This doesn't really do anything to justify the change, especially as since these are tests.